From ee295ae46f0d92e902d818e7a7207c59b828581d Mon Sep 17 00:00:00 2001
From: Valentine Barshak <vbarshak@dev.rtsoft.ru>
Date: Sat, 9 Mar 2013 04:23:24 +0400
Subject: [PATCH 044/127] arm: tegra: ktt30: Set uImage load address for KTT30

This enables UIMAGE_LOADADDR in ktt30_defconfig to support
the latest U-Boot, which is located at 0x80108000.

Signed-off-by: Valentine Barshak <vbarshak@dev.rtsoft.ru>
---
 arch/arm/configs/ktt30_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/ktt30_defconfig b/arch/arm/configs/ktt30_defconfig
index 3c721aa..c625cbd 100644
--- a/arch/arm/configs/ktt30_defconfig
+++ b/arch/arm/configs/ktt30_defconfig
@@ -49,6 +49,7 @@ CONFIG_HIGHMEM=y
 CONFIG_ARM_FLUSH_CONSOLE_ON_RESTART=y
 CONFIG_ZBOOT_ROM_TEXT=0x0
 CONFIG_ZBOOT_ROM_BSS=0x0
+CONFIG_UIMAGE_LOADADDR=y
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_USERSPACE=y
 CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
-- 
1.7.4.4

